ca/ca.go

```diff
@@ -237,10 +223,9 @@ func makeCertificateProfilesMap(defaultName string, profiles map[string]*issuanc
 		}
 
 		profilesByName[name] = &withID
-
 		_, found := profilesByHash[hash]
 		if found {
-			return certProfilesMaps{}, fmt.Errorf("duplicate certificate profile hash %d", hash)
+			return certProfilesMaps{}, fmt.Errorf("duplicate certificate profile hash %d", profile.Hash())
```

Comment:
```suggestion
			return certProfilesMaps{}, fmt.Errorf("duplicate certificate profile hash %d", hash)
```

Or don't bother with `hash := profile.Hash()` up on line 217.